DigitalLearningSolutions.Data/Models/DelegateUpload/DelegateTableRow.cs

```diff
@@ -143,5 +187,12 @@ public bool MatchesDelegateUser(DelegateUser delegateUser)
 
             return (delegateUser.EmailAddress ?? string.Empty) == Email;
         }
+
+        private bool IsEmailFormatValid()
+        {
+            var whitespace = Email!.Any(char.IsWhiteSpace);
+            var index = Email!.IndexOf('@');
+            return !whitespace && index > 0 && index != Email.Length - 1 && index == Email.LastIndexOf('@');

```

Comment:
Could we just do "new EmailAddressAttribute().IsValid(Email);" instead of rewriting the logic around "@"?

Reply:
I did exactly that at first but I thought you would say it's incorrect/unintended usage of an "attribute" 🤦‍♂️